### PR TITLE
implement cors handler

### PR DESCRIPTION
**Pull request by Julia-Protasevich**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -16,9 +17,11 @@ databaseConnect();
 
 app.use(passport.initialize()); 
 
+
 app.use('/books', booksRouter);
 app.use('/user', usersRouter);
 
+corsSetup(app);
```
Comment (MihailMartinovich) [inline app.js:24]: Have you checked that it is working?

Comment (Julia-Protasevich) [reply app.js:24]: I checked it with Postman OPTIONS request. The response contained exactly the amount of methods I have defined in my CORS parameters.

Comment (MihailMartinovich) [reply app.js:24]: but have you checked that your existing api works with this options besides OPTIONS request?

Comment (Julia-Protasevich) [reply app.js:24]: Yes, I have and it does work as it used to.

Comment (Julia-Protasevich) [reply app.js]: Sorry, I was wrong, you are the best, I'm noob

**Pull request merged**